tests/TestUtils/BaseTestCase.py

```diff
@@ -66,6 +66,15 @@ def setup_for_chrome(self, mode):
         chrome_options.add_argument(mode)
         chrome_options.add_argument("--no-sandbox")
         chrome_options.add_argument("window-size=1200x600")
+        # Set up download directory
+        # https://stackoverflow.com/questions/18437816/how-to-find-chrome-download-path-in-java
```

Comment:
Should I leave the links in for future reference?

Reply:
I think leaving the links will be fine... Eventually if we make documentation for writing tests, we could move these links over to that.